bundles/org.openhab.binding.hdpowerview/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -47,16 +55,28 @@
 	<channel-type id="scene-activate">
 		<item-type>Switch</item-type>
 		<label>Activate</label>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Mode</tag>
```

Comment:
I'm not convinced this can be considered as a "mode". Can you point me to some documentation for this tag? The scene channels are stateless and used to trigger a scene that will probably change some blind positions, vanes etc.